components/input/input.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Input Component</title>
+    <link rel="stylesheet" href="input.css" />
+  </head>
+  <body>
+    <div class="flex-container-vertical">
+        <h3>Input Component</h3>
+        <ul>
+            <li>Normal input</li>
+            <li>Input with success validation</li>
+            <li>Input with warning validation</li>
+            <li>Input with error validation</li>
+            <li>Text area</li>
+        </ul>
+
+      <form>
+        <label>First name </label>
+        <br />
+        <input />
+        <br />
+        <br/>
```

Comment:
don't use break like this
you can make this form a flex and use gap
or simply wrap label and input with div and give it margin
do correct me if I am wrong

Reply:
Hi Mimansha, Thanks for reviewing. And yea, flex works. I will make the suggested changes.